typescript/looker/rtl/browserTransport.ts

```diff
@@ -77,24 +79,26 @@ export class BrowserTransport extends BaseTransport {
 
   private async initRequest(
     method: HttpMethod,
+    path: string,
     body?: any,
     authenticator?: Authenticator,
     options?: Partial<ITransportSettings>,
   ) {
     options = options ? {...this.options, ...options} : this.options
-    let headers: {[key:string]: string} = {'x-looker-appid': agentTag}
+    let headers: IRequestHeaders = {'x-looker-appid': agentTag}
     if (options && options.headers) {
       Object.keys(options.headers).forEach(key => {
         headers[key] = options!.headers![key]
       })
     }
 
     let props = {
+      url: path,
       body: body ? JSON.stringify(body) : undefined,
       headers: headers,
       credentials: 'same-origin',
       method,
-    }
+    } as IRequestProps
```

Comment:
Good call. In some cases, using `as` will allow you to omit properties that would otherwise be required with an explicit typedef, but it's not required in this instance.